server/lomas_server/administration/README.md

```diff
@@ -0,0 +1,159 @@
+  - `-od, --overwrite_datasets`: Overwrite the existing datasets collection (optional, default: False).
+  - `-om, --overwrite_metadata`: Overwrite the existing metadata collection (optional, default: False).
+- `del_dataset`: Delete dataset and metadata from datasets and metadata collection.
+  - `-d, --dataset`: Name of the dataset to be deleted (required).
+- `show_dataset`: Show a dataset from the dataset collection.
+  - `-d, --dataset`: Name of the dataset to show (required).
+- `show_metadata`: Show metadata from the metadata collection.
+  - `-d, --dataset`: Name of the dataset of the metadata to show (required).
+- `get_datasets`: Get the list of all datasets in the 'datasets' collection.
+
+#### Collections
+
+- `drop_collection`: Delete a collection from the database.
+  - `-c, --collection`: Name of the collection to be deleted. Choices: "users", "datasets", "metadata", "queries_archives" (required).
+- `show_collection`: Print a collection.
+  - `-c, --collection`: Name of the collection to be shown. Choices: "users", "datasets", "metadata", "queries_archives" (required).
+
+### Examples
+```
+# Add a user
+python mongodb_admin.py add_user -u username
+
+# Add a user with budget
+python mongodb_admin.py add_user_with_budget -u username -d dataset_name -e 0.5 -del 0.1
+
+# Delete a user
+python mongodb_admin.py del_user -u username
+
+# Add a dataset to a user
+python mongodb_admin.py add_dataset_to_user -u username -d dataset_name -e 0.5 -del 0.1
+
+# Delete a dataset from a user
+python mongodb_admin.py del_dataset_to_user -u username -d dataset_name
+
+# Set budget field for a user and dataset
+python mongodb_admin.py set_budget_field -u username -d dataset_name -f initial_epsilon -v 0.5
+
+# Set may query field for a user
+python mongodb_admin.py set_may_query -u username -v True
+
+# Show user metadata
+python mongodb_admin.py show_user -u username
+
+# Create users collection from a YAML file
+python mongodb_admin.py add_users_via_yaml -yf users.yaml -c
+
+# Show all previous queries from user "username"
+python mongodb_admin.py show_archives -u username
+
+# Get the list of all users
+python mongodb_admin.py get_users
+
+# Get the list of all datasets from user "username"
+python mongodb_admin.py get_user_datasets -u username
+
+# Add a dataset
+python mongodb_admin.py add_dataset -d dataset_name -db database_type -d_path dataset_path -m_db metadata_database_type
+
+# Create datasets from a YAML file
+python mongodb_admin.py add_datasets_via_yaml -yf datasets.yaml -c -od -om
+
+# Delete a dataset
+python mongodb_admin.py del_dataset -d dataset_name
+
+# Show dataset "dataset_name"
+python mongodb_admin.py show_dataset -d dataset_name
+
+# Show metadata for dataset "dataset_name"
+python mongodb_admin.py show_metadata -d dataset_name
+
+# Drop a collection
+python mongodb_admin.py drop_collection -c users
+
+# Show a collection
+python mongodb_admin.py show_collection -c datasets
```

Comment:
it is `mongodb_admin_cli.py`, not `mongodb_admin.py` anymore (you can copy paste from the doc)

I had removed the readme to avoid duplicating the doc but I agree it is ok to aslo have it here

Reply:
Yes, adding it back was a mistake on my side during git conflict resolving..